scylladb/sphinx-scylladb-theme
# remove: font awesome

Depends on https://github.com/scylladb/sphinx-scylladb-theme/pull/1210

Removes font awesome support now that we have a custom font lib for icons.

## How to test

1. Build the docs
2. Make sure all icons render as expected.
